### PR TITLE
LRDOCS-7683 using React, Angular, and Vue JS

### PR DESCRIPTION
**Pull request by mwilliams2014**
@jrhoun here are the docs for migrating React, Angular, and VueJS apps to Portal. I'll send the Adapting portlets docs in a separate PR.

### CODE REVIEW

File: docs/dxp/7.x/en/developing-applications/developing-a-single-page-application/using-angular.md [hunk 1]
```diff
@@ -1,3 +1,282 @@
 # Using Angular
 
-Coming soon!
+> Liferay DXP 7.1+
+
+Running an existing Angular app on @product@ makes the app available as a widget for using on site pages. You can [adapt your existing Angular app](TODO), but this doesn't give you access to the bundler and its various loaders to develop your project further. To have access to all of Liferay DXP's features, you must use the Liferay JS Generator and Liferay npm Bundler to merge your files into a portlet bundle, adapt your routes and CSS, and deploy your bundle.
```
Comment (jrhoun) [inline docs/dxp/7.x/en/developing-applications/developing-a-single-page-application/using-angular.md]: Can you update the `@product@` tokens?

Comment (mwilliams2014) [reply docs/dxp/7.x/en/developing-applications/developing-a-single-page-application/using-angular.md]: Thought I replaced them all. Yeah I'll take care of it and push up the changes.

Comment (mwilliams2014) [reply docs/dxp/7.x/en/developing-applications/developing-a-single-page-application/using-angular.md]: Updated @jrhoun

**Comment by jrhoun** [comment]
Hey Mike - 

Sorry to inundate you with a bunch of review feedback! Can you review this branch again and update the samples similar to how we refactored the examples for [fragments](https://github.com/liferay/liferay-learn/blob/master/docs/dxp/7.x/en/site-building/developer-guide/developing-page-fragments/creating-a-contributed-fragment-collection.md)? And can you also send this one along by @sez11a as well? Thanks! (and sorry!) 😅

**Comment by mwilliams2014** [comment]
@jrhoun  I'll do my best in the time I have left.

**Comment by mwilliams2014** [comment]
Sent https://github.com/sez11a/liferay-learn/pull/105

**Pull request closed without merging**